src/provider/websocket/ws.ts

```diff
@@ -128,8 +128,8 @@ export class WSProvider implements Provider {
    */
   waitForOpenConnection = (): Promise<null> => {
     return new Promise((resolve, reject) => {
-      const maxNumberOfAttempts = 10;
-      const intervalTime = 200; //ms
+      const maxNumberOfAttempts = 50;
+      const intervalTime = 200; // ms
```

Comment:
Context switching to this interval seems excessive. I'd use something like 20 attempts at 0.5s instead

Reply:
We still keep the `10s` timeout, but don't spam it as much. I like it :)

Changed:
[58e63f8](https://github.com/gnolang/tm2-js-client/pull/85/commits/58e63f843d0016c3e652627294f949e25739628f)